docs/guides/getting-started-orchestrate-human-tasks.md

```diff
@@ -23,21 +25,26 @@ import RunProcessSM from './img/run-process-sm.png';
 import OperateHumanTasks from './img/operate-human-tasks.png';
 import FormEditorImg from './img/form-editor.png';
 import NavigationHistoryImg from './img/modeler-navigation-history.png';
+import PlayButtonImg from './img/play-play-button.png';
+import PlayOpenFormImg from './img/play-open-form.png';
+import PlayChickenImg from './img/play-chicken-complete.png';
 
 import clsx from "clsx";
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 import SaasPrereqs from './react-components/saas-prerequisites.md'
 import Install from './react-components/install-c8run.md'
 
-Camunda 8 allows you to orchestrate processes with human tasks of any complexity. Utilizing user tasks, you can create and assign tasks to users. Then, users can perform their work and enter the necessary data to drive the business process.
+You can use Camunda 8 to orchestrate processes with human tasks of any complexity. User tasks allow you to create and assign tasks to users, who can then perform their work and enter the required data to drive the business process.
```

Comment:
```suggestion
You can use Camunda 8 to orchestrate processes with human tasks of any complexity. Use the user task BPMN element to create and assign tasks to users, who can then perform their work and enter the required data to drive the business process.
```
For the getting started with HTO guide, I'd also be interested in seeing @volodymyr-melnykc 's perspective as he's the HTO PM